test/functional/license_test.rb

```diff
@@ -0,0 +1,150 @@
+require 'functional/helper'
+require 'tmpdir'
+require 'yaml'
+
+describe 'The license acceptance mechanism' do
+  include FunctionalHelper
+
+  describe 'when the license has not been accepted' do
+    describe 'when the user passes the --accept-license flag' do
+      it 'should silently work normally' do
+        Dir.mktmpdir do |tmp_home|
+          run_result = run_inspec_process('shell -c platform.family --accept-license', env: { 'HOME' => tmp_home })
```

Comment:
The command line arg has changed to `--chef-license accept` or `--chef-license accept-no-persist`

Reply:
Thanks, updated